heron/uploaders/src/java/com/twitter/heron/uploader/s3/S3Uploader.java

```diff
@@ -199,7 +199,7 @@ public URI uploadPackage() throws UploaderException {
       return resourceUrl.toURI();
     } catch (URISyntaxException e) {
       throw new UploaderException(
-          String.format("Could not convert URL %s to URI", resourceUrl, e));
+          String.format("Could not convert URL %s to URI", resourceUrl));
```

Comment:
You correctly found a bug though, good catch. Would be a gidd fix in another pr:
```
throw new UploaderException(
  String.format("Could not convert URL %s to URI", resourceUrl), e);
```